### ISSUE TITLE
I Love your string.h & object.h, but we have to make a choice.

### ISSUE THREAD

**Issue by drakstik** [bacigaluped/assignment1#18]
To everybody implementing this Map:

We have a choice to make, so please reply or like this comment and make your opinion heard. Which object & string interfaces do we want to use?

1. Professor Jan's https://piazza.com/class/k51bluky59n2jr?cid=166
2. This Guy's: https://piazza.com/class/k51bluky59n2jr?cid=345

I prefer Jan's because it is clean and comprehensive. The two are not too different, but let's make that choice now guys. Thanks.

**Comment by dermers** [comment]
The professor's has a few known problems and Professor Jan even mentioned in class today that he does not officially endorse those, we are just welcome to use them.

Unfortunately however, I would oppose both. I made PRs attempting to normalize Object and String Sunday night. They were closed with no comment yesterday, but my other spec teams happily merged them. I am not going to make them do so again.
